api/app/tests/hfi/test_pdf_gen.py

```diff
@@ -18,3 +21,24 @@ def test_gen_daily_pdf():
             fire_centres.append(fc)
         pdf_bytes = generate_daily_pdf(HFIResultResponse(**result), fire_centres)
         assert len(pdf_bytes) > 0
+
+
+def test_gen_prep_data_converter():
+    """ All dailies in prep pdf data are grouped by station code and sorted by date"""
+    with open(test_hfi_result, 'r') as hfi_result:
+        result = json.load(hfi_result)
+        prep_pdf_data = response_2_prep_cycle_jinja_format(HFIResultResponse(**result))
+
+        # List of dates for prep period
+        dates = []
```

Comment:
Let's add this to the `response_2_prep_cycle_jinja_format` function otherwise we'll only have formatted dates in this test